lib/saml/url_service.rb

```diff
@@ -14,6 +14,8 @@ class URLService
       'http://127.0.0.1:3000' => { base_redirect: 'http://127.0.0.1:3001' }
     }.freeze
 
+    SSOE_LOGOUT_URL = 'https://int.eauth.va.gov/pkmslogout'
```

Comment:
Probably deserves to be a `Settings.yml` config instead of hardcoding it in `url_service.rb`.